src/components/forms/Form/Form.stories.tsx

```diff
@@ -174,3 +175,45 @@ export const addressForm = (): React.ReactElement => (
     </Fieldset>
   </Form>
 )
+
+export const signInForm = (): React.ReactElement => (
+  <Form onSubmit={mockSubmit} large>
+    <Fieldset legend="Sign In">
+      <span>
+        or <a href="javascript:void(0);">create an account</a>
+      </span>
+
+      <Label htmlFor="username">Username or email address</Label>
+      <TextInput id="username" name="username" type="text" />
+      <Label htmlFor="password-sign-in">Password</Label>
+      <TextInput
+        id="password-sign-in"
+        name="password-sign-in"
+        type="password"
+      />
+    </Fieldset>
```

Comment:
Do you mind adding the "Show password" text & submit button? IMO we don't have to implement the show password toggle functionality, unless you'd like to..

Reply:
@suzubara Yes! I was confused if I should include those.